### ISSUE TITLE
[BUG] PowerShell startup issue that persists after reset

### ISSUE THREAD

**Issue by kobulloc-MSFT** [Azure/CloudShell#219]
## To Reproduce

This issue persists after creating a new storage account (see screenshot).

1. Click on gear setting
2. Click on Reset user settings
3. Click on Reset to confirm reset
4. Select PowerShell at welcome prompt
5. Select subscription from dropdown
6. Click on Create storage

## Observed Behavior

PowerShell immediately errors out on start (reproducible and consistent over a couple months) :

![image](https://user-images.githubusercontent.com/37556655/179852041-865768f8-176e-41f4-9dd2-5488c6c07603.png)

## Expected behavior

Expected that PowerShell returns a prompt with no error on start.

## Is this specific to Cloud Shell?
This is only encountered in Azure Cloud Shell when running PowerShell (also seen in Bash when running "pwsh").

## Interface information
How are you accessing Cloud Shell - https://ms.portal.azure.com/#cloudshell/
If a browser, which Operating System and browser are you using? Chrome on Windows 10

**Comment by maertendMSFT** [comment]
Duplicate of: https://github.com/Azure/CloudShell/issues/175

**Comment by kobulloc-MSFT** [comment]
@maertendMSFT I'm still seeing this issue after doing a reset:

![image](https://user-images.githubusercontent.com/37556655/187544849-f0663b9e-658a-4236-9b75-1c329c6396fd.png)

![image](https://user-images.githubusercontent.com/37556655/187544937-71a3800b-73a9-4655-8b81-e54f4312767b.png)

**Comment by maertendMSFT** [comment]
@kobulloc-MSFT, have you only tried resetting your settings via the option in the toolbar, or have you also run the `clouddrive unmount` command?

if you havent tried the command, can you do so and use a fresh storage account?

**Comment by kobulloc-MSFT** [comment]
@maertendMSFT I just ran the [`clouddrive unmount`](https://docs.microsoft.com/en-us/azure/cloud-shell/persisting-shell-storage#unmount-clouddrive) command and I'm seeing the same errors after following the prompts and creating a new storage account.

**Comment by robins1212** [comment]
@kobulloc-MSFT just to confirm when you reset your user settings and started over you're using the same subscription as before and no advanced settings? If so, Cloud Shell is reusing the same storage account and fileshare as before which is why unmounting and resetting your settings is not successful. Can you delete the fileshare from your storage account and try relaunching?

**Comment by kobulloc-MSFT** [comment]
@robins1212 That appears to have worked.  Thank you!

**Steps:**
1. From the Azure portal I went to Services > Storage accounts.
2. Within my subscription I located a storage account that had been automatically created on my behalf for Azure Cloud Shell.  This storage account had a seemingly randomly generated name that started with "cs" (example: cs123456ab7c9012d) and belonged to the "cloud-shell-storage-regionname" resource group.
3. I deleted that storage account.
4. Going back to Azure Cloud Shell, I was automatically prompted to complete the reset user settings wizard.  I completed the wizard using default values (I did not use advanced settings).
5. PowerShell is now working and the same seemingly random storage account was created using an identical name.
